enable building WXT terminal by setting WXT=1 in makefile configuration section

diff --git a/config/makefile.mgw b/config/makefile.mgw
--- a/config/makefile.mgw
+++ b/config/makefile.mgw
@@ -92,9 +92,9 @@
 # Requires wxWidgets, Cairo, Pango, and their respective dependencies.
 # Uncomment the following lines to compile it.
 # WXT_LOCATION should give the path where can be found wx/*.h headers
-#WXT=1
+WXT=1
 #WX_LOCATION=/usr/local/include/wx-2.6
-#CAIROLIBS=1
+CAIROLIBS=1
 
 # Uncomment the following line to select the lua/tikz terminal
 #LUA=1
